src/routes/components/TextField.tsx

```diff
@@ -554,22 +554,253 @@ export default function TextFieldPage() {
             </Sandbox>
 
             <h3 id="component-example-phone">Phone number</h3>
-            <Sandbox flags={["reactive"]}>
+            <Sandbox flags={["reactive"]} skipRender>
+              {/*CSS Code Snippet*/}
+              <CodeSnippet
+                lang="css"
+                allowCopy={true}
+                code={`
+                  .medium-text {
+                    font: var(--goa-typography-body-m);
+                  }
+                `}
+              />
+              {/*Angular Code Snippet - need for leadingContent slot*/}
+              <CodeSnippet
+                lang="typescript"
+                tags="angular"
+                allowCopy={true}
+                code={`
+                  export class ExampleComponent {
+                    value= "";
+                    
+                    onChange(event: Event) {
+                      this.value = (event as CustomEvent).detail.value;
+                    }
+                  }
+                `}
+              />
+              <CodeSnippet
+                lang="html"
+                tags="angular"
+                allowCopy={true}
+                code={`
+                  <goa-form-item label="Phone number">
+                    <goa-input (_change)="onChange($event)" [value]="value" name="phone">
+                      <div slot="leadingContent">
+                        <goa-block direction="row" alignment="center" gap="none">
+                          <goa-icon type="add" size="small"></goa-icon> <span class="medium-text">1</span>
```

Comment:
Maybe just use `+1` instead of the icon. The `medium-text` styling also increases the height of the input by a couple pixels, so maybe it should be removed as well.

Below is how it looks without the icon and the `medium-text` class, which is fine.
![image](https://github.com/GovAlta/ui-components-docs/assets/41582/09dc8eb1-c49e-48bf-b926-8fee8f366acf)

Reply:
The issue is that this example being shown isn't something that we would recommend other teams to do as it doesn't make sense to use an icon in place of an existing character, as well as when looking at this example it isn't immediately obvious that it isn't just text. A better example would be to use a single icon i.e telephone. This also allows use to remove the `.medium-text` class, which just adds noise to the example.

Reply:
Then is this whole example not required?